examples/ping.rs

```diff
@@ -10,6 +10,7 @@ mod utils;
 use std::str::FromStr;
```

Comment:
Do you think you could fold this commit into previous ones? Same reasoning as before.